internal/idle/idle_e2e_test.go

```diff
@@ -83,26 +86,43 @@ func channelzTraceEventFound(ctx context.Context, wantDesc string) error {
 	return fmt.Errorf("when looking for channelz trace event with description %q, %w", wantDesc, ctx.Err())
 }
 
-// channelzTraceEventNotFound looks up the top-channels in channelz (expects a
-// single one), and verifies that there is no trace event on the channel
-// matching the provided description string.
-func channelzTraceEventNotFound(ctx context.Context, wantDesc string) error {
-	sCtx, sCancel := context.WithTimeout(ctx, defaultTestShortTimeout)
-	defer sCancel()
+// Registers a wrapped round_robin LB policy for the duration of this test that
+// retains all the functionality of the round_robin LB policy and makes the
+// balancer close  event available for inspection by the test.
+//
+// Returns a channel that gets pinged when the balancer is closed.
+func registerWrappedRoundRobinPolicy(t *testing.T) chan struct{} {
+	rrBuilder := balancer.Get(roundrobin.Name)
+	internal.BalancerUnregister(rrBuilder.Name())
```

Comment:
This is unnecessary.  Just register the new one over the old one.

Reply:
Done.